packages/good-design/src/core/layout/BalanceGD.tsx

```diff
@@ -59,4 +59,72 @@ const BalanceGD: FC<BalanceGDProps> = ({ gdPrice, requiredChainId, refresh = "ne
   );
 };
 
+const abbreviateGd = (numberStr: string, decPlaces = 2) => {
```

Comment:
i believe there's already a utility we use for that

Reply:
The only thing I found was in the wallet which used an external library 'numeral' which I found a bit of an overkill